Client/game_sa/CModelInfoSA.cpp

```diff
@@ -256,23 +256,7 @@ BOOL CModelInfoSA::IsVehicle ( )
 
 bool CModelInfoSA::IsPlayerModel()
 {
-    return (m_dwModelID == 0 ||
-        m_dwModelID == 1 ||
-        m_dwModelID == 2 ||
-        m_dwModelID == 7 ||
-        (m_dwModelID >= 9 &&
-            m_dwModelID != 208 &&
-            m_dwModelID != 149 &&
-            m_dwModelID != 119 &&
-            m_dwModelID != 86 &&
-            m_dwModelID != 74 &&
-            m_dwModelID != 65 &&
-            m_dwModelID != 42 &&
-            m_dwModelID <= 272) ||
-            (m_dwModelID >= 274 &&
-                m_dwModelID <= 288) ||
-                (m_dwModelID >= 290 &&
-                    m_dwModelID <= 312));
+    return ( GetInterface() && GetInterface()->pColModel && GetInterface()->pColModel == ( CColModelSAInterface * ) VAR_CTempColModels_ModelPed1 );

```

Comment:
Looks like there is only one collision? Seems weird...

http://gtaforums.com/topic/809134-restoring-the-pedscol/